src/Microsoft.AspNetCore.Mvc.ViewFeatures/Rendering/TagBuilder.cs

```diff
@@ -343,6 +329,110 @@ private string DebuggerToString()
             }
         }
 
+        /// <summary>
+        /// Returns the <see cref="IHtmlContent"/> body.
+        /// </summary>
+        /// <returns>The inner <see cref="IHtmlContent"/>.</returns>
+        public IHtmlContent RenderBody()
+        {
+            return _innerHtml;
```

Comment:
@sebastienros no need. Everything that consumes `IHtmlContent` does the right thing (nothing) w/ `null`.